SDL/MiniDoublet.cuh

```diff
@@ -73,61 +53,901 @@ namespace SDL
         float* outerLowEdgeX;
         float* outerLowEdgeY;
 
-#ifdef CUT_VALUE_DEBUG
-        //CUT VALUES
-        float* dzCuts;
-        float* drtCuts;
-        float* drts;
-        float* miniCuts;
-#endif
```

Comment:
It's been a while since I moved this file over, but looking at the code I don't think any of these variables are ever used in the code. I checked for occurrences of each but can't find anything, so these must be left over on the master branch from a previous PR. I think that's why I removed them from here. @YonsiG are these variables okay to remove? It seems like you commented out their definitions in Minidoublet.cu around a year ago but left them intact here.

Reply:
I'm not sure if those variables are valid for use now. Last time, when I was trying to use those variables and compile with "-d", which means defining CUT_VALUE_DEBUG, it couldn't function correctly at least in the Minidoublet.cu, so I comment out that part. If this is not used anywhere else, I believe we can remove it.

Reply:
Sounds good, thank you Yonsi